packages/flutter_tools/templates/plugin_ffi/macos.tmpl/Classes/projectName.c.tmpl

```diff
@@ -0,0 +1,2 @@
+// Relative import to be able to reuse the C sources.
```

Comment:
It's probably worth explicitly pointing to the podfile to explain why (e.g., "See the comment in [...].podfile for more information). Otherwise people may be confused if/when they want to add more sources, since they are much more likely to look at this file than the podfile.